docs/admin/postgres.mdx

```diff
@@ -23,19 +22,23 @@ intarray
 pg_stat_statements
 pg_trgm
 pgcrypto
+vector
+pg_vector
 ```
 
 ## [Configuring PostgreSQL](/admin/config/postgres-conf)
 
-## Upgrading PostgreSQL
+### Upgrading Built-in PostgreSQL
 
-Sourcegraph uses PostgreSQL as its main database and this documentation describes how to upgrade PostgreSQL between major versions.
+Sourcegraph deployments come packaged with containers using the `postgresql-16` image for `pgsql` and `codeintel-db` deployments and `postgresql-16-codeinsights` for `codeinsights`. These images contain an entryscript which will detect and upgrade postgres instances below version 12. See the (postgres 12 end of life notice)[/admin/prostgres12_end_of_life_notice#] for more details. 
```

Comment:
minor: I believe it would only support upgrading from 12, not anything below 12?

Reply:
> minor: I believe it would only support upgrading from 12, not anything below 12?

Yeah, that's correct. I don't _think_ we have any customers running anything older than 12 but we should be clear 😄